src/main.cpp

```diff
@@ -223,7 +224,7 @@ int main( int argc, char *argv[]) {
                     std::cout << "Toggle File Loop" << std::endl;
                     song.setMode(LOOP_FILE);
                 } else {
-                    mq.SendMessage("Invalid");
+                    mq_to_client.SendMessage("Invalid");
                 }
                 //Delay to prevent sending message to self
                 std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
I don't think we need the delay anymore. You could probably remove this line and amend the changes to this commit.